docs/jenkins.md

```diff
@@ -126,13 +126,11 @@ The output consists of the following:
 - An overview of all ratings, compared against the system as a whole. This allows you to check if your changes improved the system, or accidentally made things worse.
 - The final conclusion on whether your changes and merge request meet the quality target.
 
-In addition to the textual output, Sigrid CI also generates a static HTML file that shows the results in a more graphical form. This is similar to test coverage tools, which also tend to produce a HTML report. The information in the HTML report is based on the aforementioned list, though it includes slightly more detail.
+In addition to the textual output, the results also include a link to the Sigrid landing page. 
 
-<img src="images/feedback-report.png" width="600" />
+<img src="images/landing-page.png" width="700" />
 
-You might notice at this point that the report does not contain any styling, and is instead shown as black text on white background. If so, this is caused by your Jenkins configuration disallowing styling in build artifacts. You can enable this as described in the [Jenkins documentation](https://www.jenkins.io/doc/book/security/configuring-content-security-policy/). The Sigrid CI report only contains inline CSS, so the correct value to enable the styling is `default-src 'none'; img-src 'self'; style-src 'self' 'unsafe-inline';`. 
-
-Finally, if you want to have more information on the system as a whole, you can also access [Sigrid](http://sigrid-says.com/), which gives you more information on the overall quality of the system, its architecture, and more.
+You can open this URL in order to use Sigrid for interpreting your analysis results. Whether you should use the text output or the Sigrid page is largely down to personal preference: the text output is faster to acces and more concise, while Sigrid allows you to view results in a more visual and interactive way. 
```

Comment:
The link to the Sigrid landing page is technically part of the textual output (contrary to what was mentioned in the old text). I wanted to suggest adding it to the list as another bullet but I think it's clearer to indeed have this text separated from the list. I would however change the text accordingly. I would also move the first sentence of the next paragraph here. The result would be the following:

```
At the end of the textual output there is a link to the Sigrid landing page. You can open this URL in order to use Sigrid for interpreting your analysis results.

<img src="images/landing-page.png" width="700" />

Whether you should use the text output or the Sigrid page is largely down to personal preference: the text output is faster to acces and more concise, while Sigrid allows you to view results in a more visual and interactive way. 
```

(I can't seem to put this as a suggestion directly as GitHub won't allow those on removed lines and even in split mode I can't only select the new and unchanged lines...)